### PR TITLE
navbar and audioplayer empty placeholder

### PR DESCRIPTION
**Pull request by muhajirdev**
### Summary
TSIA

### CODE REVIEW

File: src/components/AudioPlayer/AudioPlayer.module.scss [hunk 1]
```diff
@@ -2,6 +2,10 @@
 @use "src/styles/constants";
 @use "src/styles/utility";
 
+.emptySpacePlaceholder {
```
Comment (tifa2UP) [inline src/components/AudioPlayer/AudioPlayer.module.scss:5]: Might be good to add a comment here explaining why we have this placeholder

File: src/components/AudioPlayer/AudioPlayer.module.scss [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+  height: calc(1 * constants.$audio-player-default-height);
```
Comment (tifa2UP) [inline src/components/AudioPlayer/AudioPlayer.module.scss:6]: height: constants.$audio-player-default-height;

Reminds me of this meme 😂 

image

File: src/components/AudioPlayer/AudioPlayer.tsx [hunk 3]
```diff
@@ -86,59 +86,62 @@ const AudioPlayer = () => {
...
-          <AudioRepeatManager
-            audioPlayerElRef={audioPlayerElRef}
-            reciterId={reciterId}
-            chapterId={audioData?.chapterId}
+          <AudioKeyBoardListeners
+            seek={(seekDuration) => {
+              triggerSeek(seekDuration);
+            }}
+            togglePlaying={() => togglePlaying()}
+            isAudioPlayerHidden={isHidden}
```
Comment (tifa2UP) [inline src/components/AudioPlayer/AudioPlayer.tsx:110]: Is this intentional?

File: src/components/QuranReader/QuranReader.module.scss [hunk 4]
```diff
@@ -2,8 +2,8 @@
 @use "src/styles/breakpoints";
 @use "src/styles/utility";
 
-$quran-reader-padding-mobile: 4rem;
-$quran-reader-padding-tablet: 5rem;
+$quran-reader-padding-mobile: 2rem;
+$quran-reader-padding-tablet: 3rem;
```
Comment (tifa2UP) [inline src/components/QuranReader/QuranReader.module.scss:6]: nice

File: src/components/AudioPlayer/AudioPlayer.module.scss (hunk 1 above)
Comment (tifa2UP) [reply src/components/AudioPlayer/AudioPlayer.module.scss:5]: nit: I would call this heightPlaceholder. but not a strong opinion. up to you :)

**Review by tifa2UP — APPROVED** [review]

File: src/components/AudioPlayer/AudioPlayer.module.scss (hunk 2 above)
Comment (muhajirdev) [reply src/components/AudioPlayer/AudioPlayer.module.scss:6]: lol

File: src/components/AudioPlayer/AudioPlayer.tsx [hunk 5]
```diff
@@ -86,59 +86,62 @@ const AudioPlayer = () => {
...
-        />
-        <div className={styles.sliderContainer}>
-          <AudioPlayerSlider
-            audioPlayerElRef={audioPlayerElRef}
-            isMobileMinimizedForScrolling={isMobileMinimizedForScrolling}
+          {reciterId && audioData?.chapterId && (
+            <QuranReaderHighlightDispatcher
+              audioPlayerElRef={audioPlayerElRef}
+              reciterId={reciterId}
+              chapterId={audioData?.chapterId}
```
Comment (muhajirdev) [inline src/components/AudioPlayer/AudioPlayer.tsx:116]: a

File: src/components/AudioPlayer/AudioPlayer.tsx (hunk 3 above)
Comment (muhajirdev) [reply src/components/AudioPlayer/AudioPlayer.tsx:110]: image

seems to be something to do with Github diff things. looks fine on vscode

**Pull request merged**